Sources/KukaiCoreSwift/Models/Token.swift

```diff
@@ -139,3 +139,12 @@ extension Token: Hashable {
 		hasher.combine(tokenContractAddress)
 	}
 }
+
+extension Token: Identifiable {
+
+	/// Conforming to `Identifiable` to enable working with ForEach and similiar looping functions
+	/// `tokenContractAddress` will return empty for `XTZ` so we'll just use `tokenType` because it represents XTZ for my use case
+	public var id: String {
+		tokenContractAddress ?? tokenType.rawValue
```

Comment:
@aklowther you'll need to include tokenID here as well. There are many FA2 tokens that are created with the same token and differ only with token id. The tezotopia resource tokens for example (UNO, ENR etc) are all the same with only a difference in token ID